rest/src/db/CatapultDb.js

```diff
@@ -423,6 +432,57 @@ class CatapultDb {
 			})));
 	}
 
+	/**
+	 * It retrieves and adds the blocks information to the transactions' meta.
+	 *
+	 * The block information includes its timestamp and feeMultiplier
+	 *
+	 * @param {object[]} list the list without the added block information.
+	 * @returns {Promise<object[]>} this list with the added block information.
+	 */
+	async addBlockMetaToTransactionList(list) {
```

Comment:
i think in previous review (for receipts) you used variable name page. is there a reason you are using list here instead?

Reply:
Receipt only has paginated searches, transaction has paginated searches, and by ids/hashes lists. I can swap Recipient to use lists and (re)construct the page in the main method.

Reply:
that answer makes sense, so can leave as is